### PR TITLE
feat: implement landing page

### LINKED ISSUE #1761: Implement landing page
- For colors, we stick with tailwind defaults, use whatever is closest to the mockup.
- For CSS we generally use CSS-in-JS tailwind `css=tw''`
- All CSS uses tailwind unless it is something super custom.
- All icons are fontawesome

E.g. all spacing uses tailwind, whatever is closest to mockup doesn't have to be exact.

https://www.figma.com/file/sWFy2iXpy9ixoN3L7m0N0W/Landing?node-id=2%3A2

### LINKED ISSUE #1791: Landing Application Pricing Page

### PR DESCRIPTION
**Pull request by hauleIT**
## Description (Optional)

Related #1761  
Fixes #1791 #1795

### CODE REVIEW

**Review by webberwang — CHANGES_REQUESTED** [review]

File: apps/landing/home/pricing/PricingBody.tsx [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+      'Form generation from schema',
+      'Daily backup with restore',
+    ],
+  },
+  {
+    title: 'Business',
+    price: '$149',
+    type: 'Demo Product',
+    benefit: '2 weeks trial',
+    bgColor: '#F3ECFF',
```
Comment (webberwang) [inline apps/landing/home/pricing/PricingBody.tsx:47]: these color we shouldn't hardcode, use a tailwind preset

**Review by webberwang — CHANGES_REQUESTED** [review]
check out the ci lint, one of the commit message had improper format

File: apps/landing/home/pricing/PricingBody.tsx [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Button, Col, Row } from 'antd'
+import * as React from 'react'
+import tw from 'twin.macro'
+
+const generateTextColor = (key: string) => {
+  switch (key) {
+    case 'textWhite':
+      return tw`text-white `
+    case 'textViolet700':
```
Comment (webberwang) [inline apps/landing/home/pricing/PricingBody.tsx:12]: why not just inline this in the object

**Review by webberwang — CHANGES_REQUESTED** [review]

File: apps/landing/home/pricing/PricingBody.tsx (hunk 2 above)
Comment (hauleIT) [reply apps/landing/home/pricing/PricingBody.tsx:12]: I tried, but twin.marco can not read property 'replace'

Comment (hauleIT) [reply apps/landing/home/pricing/PricingBody.tsx:12]: It's just using String

**Pull request merged**